tests/scripts/check-python-files.sh

```diff
@@ -28,6 +28,39 @@ else
     PYTHON=python
 fi
 
+can_pylint () {
+    # Pylint 1.5.2 from Ubuntu 16.04 is too old.
+    # Pylint 1.8.3 from Ubuntu 18.04 passed on the first commit containing this line.
+    $PYTHON -m pylint 2>/dev/null --version | awk '
```

Comment:
What about to check the version only with Python code ? If it is about as simple as with `awk` I'd prefer not using `awk`.